src/Controller/Admin/DashboardController.php

```diff
@@ -22,10 +23,15 @@
 
 class DashboardController extends AbstractDashboardController
 {
+    public const string MODEL_TIMEZONE = 'UTC';
+    public const string VIEW_TIMEZONE = 'Europe/Copenhagen';
+
     // Default date time format used in the UI.
     //
     // @see https://unicode-org.github.io/icu/userguide/format_parse/datetime/#datetime-format-syntax
-    public const DATETIME_FORMAT = 'dd-MM-Y HH:mm:ss';
+    public const string DATETIME_FORMAT = 'dd-MM-Y HH:mm:ss';
+    public const string TIME_FORMAT = 'HH:mm:ss';
+    public const string DATE_FORMAT = 'dd-MM-Y';
```

Comment:
Getting the “view” and “format” values from config (`.env.local`) would be really nice, but it may require a lot of effort.

Reply:
These are unlikely to change in the applications lifetime so will leave as is for now.